### DISCUSSION TITLE
WalletConnect Cloud Sign In with Email/Password doesn't work anymore

### DISCUSSION THREAD

**Discussion by thedanielmark** [WalletConnect/walletconnect-monorepo#5289] [Developer Support]
Hello,
I have multiple projects created on my WalletConnect cloud dashboard and have been using it for the better part of a year and have no problems signing in with my email/password combo. However for the last 2 months or so, everytime I login, I see the dashboard for a split second and then it redirects back to the sign in page. I have tried making a new account with metamask sign in and it works fine. The email/password combo account alone doesn't work.
I have multiple projects on there and haven't been able to access it.

**Comment by heyitsarpit** [comment]
having the same problem, I can't create an account cause no verification email is ever sent to me. @glitch-txs please escalate this.
